functionary/core/models/package.py

```diff
@@ -7,6 +7,22 @@
 
 from core.models import Environment
 
+# Pending status represents the package is being built
+# Complete status represents the package has been successfully built at least once
+# Enabled status represents that the package is can be used by users
+# Disabled status represents that the package cannot be used by users
+PENDING = "PENDING"
```

Comment:
I don't love these being bare constants.  I can see why you moved them out here, but I'd rather they be part of some class, similar to `PARAMETER_TYPES` over in `core/utils/parameter.py`.  In cases where you're importing them elsewhere, it provides much improved context IMO.  It also lets you import a single thing and have access to all of the values.